src/layouts/header.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HEADER</title>
+  <link rel="stylesheet" href="../../assets/css/output.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css"
+    integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <link href="https://fonts.cdnfonts.com/css/open-sans" rel="stylesheet">
+
+
+
+</head>
+
+<body class="font-openSans">
+  <header class="w-full border-b border-gray-200 ">
+
+    <div class="flex justify-between items-center max-w-7xl mx-auto px-2.5 pt-2.5">
+      <div class="language flex">
+        <button onclick="changeLanguage('vi')"
+          class="lang mr-5 transition-all px-3 py-0.5 bg-black text-white rounded border-solid border">
+          VI
+        </button>
+        <button onclick="changeLanguage('en')" class="lang transition-all">
+          EN
+        </button>
+      </div>
+      <div class="user-greeting flex flex-col items-center">
+        <span data-i18n="header.greeting"></span>
+        <span>Đăng Tỉnh</span>
+      </div>
+    </div>
+
+    <div class="max-w-7xl p-2.5 pt0 border-red-400 mx-auto flex justify-between items-center">
+      <div class="logo">
+        <img src="../../assets/images/mandala-logo.jpg" alt="">
+      </div>
+
+      <ul class="nav hidden lg:flex ">
+        <li class="mr-6 transition-all hover:text-primary-color"><a data-i18n="header.home"
+            href="../../assets/locales/en.json">TRANG
+            CHỦ</i></a></li>
```

Comment:
```suggestion
        <li class="mr-6 transition-all hover:text-primary-color"><a data-i18n="header.home"
            href="../../assets/locales/en.json"></i></a></li>
```
E bỏ cách text bên trong nhé